roman-amy/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "roman-amy",
+  "version": "1.0.0",
+  "description": "lab 7",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "keywords": [
+    "node",
+    "npm",
+    "express",
+    "server"
+  ],
+  "author": "Roman and Amy",
```

Comment:
In ~the future~ you'll want to make this specific enough that people could actually find you... at least by including last names and possibly also by including GitHub handles.